### DISCUSSION TITLE
using ESP32C6 chip

### DISCUSSION THREAD

**Discussion by ElBourino** [pschatzmann/ESP32-A2DP#562] [Q&A] [answered]
Hello,
I am using this (god given) library to control an external music playing device using an ESP32 module via a phone.
All is good using a generic ESP32 dev board, but it seems that the ESP32C6 chip isn't supported by this library.

Is there a way to be able to use ESP32C6 chips (such as the one from seeed) with this library?
To be noted that I only get volume and start/stop info from the phone, no music is transmitted in this application.

Thanks and cheerio

**Comment by pschatzmann** [comment, marked as answer]
The [datasheet](https://www.espressif.com/sites/default/files/documentation/esp32-c6_datasheet_en.pdf) does not mention Classical Bluettooth, so it can't work!

**Reply by ElBourino** [reply]
Thanks for the reply!
Ah, well thats no fun... I would have though that Bluetooth 5.0 was still compatible with classic Bluetooth.
I'll try to get the the old ESP32s, but would you have an idea on how/what library i could use to get AVRCP type of commands from a phone to this ESP32C6?

**Comment by pschatzmann** [comment]
How about using Google: I found this: AVRCP is a BR/EDR profile. It is not available on LE.

**Reply by ElBourino** [reply]
Thanks, I'm not a Bluetooth expert at all. 
I did notice that, I'll continue searching for an equivalent profile/solution on LE. 

In any case thanks again for the library and if by any chance you have an idea about that I'm all ears :)

**Comment by pschatzmann** [comment]
I think Espressif is working to provide Audio functionality for BLE, but no release date has been given so far
